src/main/kotlin/eda/shoppingBasket/service/entity/Offering.kt

```diff
@@ -0,0 +1,18 @@
+package eda.shoppingBasket.service.entity
+
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.ManyToOne
+import java.util.UUID
+
+@Entity
+class Offering(
```

Comment:
Not sure how we do the connection to shoppingBasketItem. A one to one mapping doesnt work i think. So maybe a oneToMany mapping, as in multiple shoppingBasketItems(in different baskets) can reference the same offering (first one to buy will get it)

Reply:
yep, a oneToMany mapping sounds good
and this will most probably be unidirectional coz I don’t think the offering repo actually cares about what’s going on in the shoppingBasketItem repo?

Reply:
well, thinking about it, ManyToOne mapping where the shoppingBasketItem will keep the reference of the Offering sounds better?